pkg/cmd/docgen/diagrams.go

```diff
@@ -659,6 +659,22 @@ var diagramCmd = func() *cobra.Command {
 						fmt.Printf("%s: (POST REPLACE)\n\n%s\n", s.name, g)
 						return
 					}
+
+					name := strings.Replace(s.name, "_stmt", "", 1)
```

Comment:
Not convinced it's worth opening up the potential for a name collision with this, but I don't feel strongly if you find the `_stmt` suffix that bothersome.

Reply:
this is the `name` from below, i just moved it up.